File: Modul4/soal5/PRAK305-2210817310014-FatihAlifGhifari.c
#include <stdio.h>

int main ()
{
    int in,i,h,j,d,m;
    while(i<=4){
        printf("-->");
        scanf("%d", &in);
        h=in/86400;
        j=(in/3600)%24;
        m=(in/60)%60;
        d=in%60;
        if (in<=3600){
            printf("%.2d:%.2d:%.2d\n", j,m,d);
    }else if (in>3600 && in<86400){
        printf("%.2d:%.2d:%.2d\n", j,m,d);
        }
    else if (in>86400){
        printf("%d hari %.2d:%.2d:%.2d\n", h,j,m,d);
    }
    i++;
}
return 0;
}

File: Modul4/soal5/PRAK305-2210817310014-FatihAlifGhifari.py
I=0
while True:
    I=I+1
    i=int(input())
    hari=int(i/84600)
    jam=int((i/3600) % 24)
    menit=int((i/60) % 60)
    detik=int(i % 60)
    if(i<=3600):
        print("%.2d:%.2d:%.2d\n" %(jam,menit,detik))
    elif(i>=3600 and i<84600):
        print("%.2d:%.2d:%.2d\n" %(jam,menit,detik))
    elif(i>=84600):
        print("%d hari %.2d:%.2d:%.2d\n" %(hari,jam,menit,detik))
    if(I>4):
        break
